### DISCUSSION TITLE
How do I assign a new keybinding to predictive suggestion?

### DISCUSSION THREAD

**Discussion by coindegen** [zimfw/zimfw#529] [General]
Hello, I'm brand new to Zim.

I noticed I can type the right arrow to auto-complete the predictive suggestion. 
Where is this configured, and how can I set a new keybinding for it?

**Comment by antoineco** [comment]
This functionality is provided by `zsh-autosuggestions`.

Default keybindings can be set in your `.zshrc` via `bindkey`: https://github.com/zsh-users/zsh-autosuggestions#key-bindings

**Reply by coindegen** [reply]
perfect thanks, just what I needed. 🙏
